src/BloomExe/Book/Book.cs

```diff
@@ -1341,6 +1341,9 @@ private XmlElement FindPageDiv(IPage page)
 			//review: could move to page
 			var pageElement = OurHtmlDom.RawDom.SelectSingleNodeHonoringDefaultNS(page.XPathToDiv);
 			Require.That(pageElement != null,"Page could not be found: "+page.XPathToDiv);
+			if (pageElement != null)
+				pageElement.InnerXml = XmlHtmlConverter.RemoveEmptySelfClosingTags(pageElement.InnerXml);
+				
```

Comment:
The browser does not recognize the self-closing syntax, so it considers the tag to be unclosed. It never ever finds a matching close tag so the whole of the rest of the document, including all the rest of the things that should be top-level children of the list-of-pages divs, are instead treated as children of the unclosed tags.